spec/webview-spec.js

```diff
@@ -506,6 +552,17 @@ describe('<webview> tag', function () {
       })
     })
 
+    it('can disable the remote module', async () => {
+      const message = await startLoadingWebViewAndWaitForMessage(webview, {
+        preload: `${fixtures}/module/preload-disable-remote.js`,
+        src: `file://${fixtures}/api/blank.html`,
+        webpreferences: 'enableRemoteModule=no'
+      })
+
+      const types = JSON.parse(message)
```

Comment:
Probably `types` should be `typeOfRemote`.

Reply:
actually it's like this in other places, where it also represents something else

Reply:
renamed